### PR TITLE
Add perf_premium_datadisks_io testcase

### PR DESCRIPTION
**Pull request by dsrivastavv**
Migrate PERF_STORAGE-DISK-IO

### CODE REVIEW

File: microsoft/testsuites/performance/storageperf.py [hunk 1]
```diff
@@ -102,12 +102,36 @@ def perf_premium_datadisks_1024k(
...
+        description="""
+        This test case uses fio to test vm with 24 data disks.
+        """,
+        priority=3,
+        timeout=TIME_OUT,
+        requirement=simple_requirement(
+            min_core_count=72,
+            disk=schema.DiskOptionSettings(
+                disk_type=schema.DiskType.PremiumSSDLRS,
+                data_disk_iops=5000,
```
Comment (squirrelsc) [inline microsoft/testsuites/performance/storageperf.py]: set the iops as min value, so it can be more flexible.

Comment (LiliDeng) [reply microsoft/testsuites/performance/storageperf.py]: it is to test iops of P30 disk, it has the minimal requirement for the iops.

Comment (squirrelsc) [reply microsoft/testsuites/performance/storageperf.py]: Set it to a min value, not a number directly.

Comment (dsrivastavv) [reply microsoft/testsuites/performance/storageperf.py]: Updated. @squirrelsc @LiliDeng  Should we set a min value in other storage performance test cases too?

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**